projects/components/facet/bootstrap/facet-filters/facet-filters.ts

```diff
@@ -92,6 +97,8 @@ export class BsFacetFilters implements OnChanges {
                 children: children
             });
         });
+
+        this.addFacetMenu();
```

Comment:
Could this new feature be optional? would be good to have an @input() named "enableCustomization" or something, so that we don't store additional information into the user settings if we don't want to.

Reply:
Will add this, very good idea.